docs/content/docs/engines/build.md

```diff
@@ -24,9 +24,15 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-# Build From Source
+# Build from Source
 
-Clone from git, enter:
+In order to build Flink you need the source code. Either [download the source of a release]({{< downloads >}}) or [clone the git repository]({{< github_repo >}}).
+
+In addition you need **Maven 3** and a **JDK** (Java Development Kit). Flink Table Store requires **Java 8** to build.
+
+*NOTE: Maven 3.3.x can build Flink Table Store, but will not properly shade away certain dependencies. Maven 3.2.5 creates the libraries properly.*
```

Comment:
This is true for Flink -- though work is ongoing to overcome this limitation -- but is it also true for the Table Store?